src/interaction/FixedLocalTupleComListInteractionTemplate.hpp

```diff
@@ -47,7 +47,9 @@ namespace espressopp {
 	private:
 	    long unsigned int num_of_part; // total number of particles
 	    long unsigned int num_of_subchain; // total number of subchains
-	    std::map<long unsigned int, Real3D> com_origin;
+	    boost::unordered_map<long unsigned int, Real3D> com_origin;
+	    boost::unordered_map<long unsigned int, Real3D> com_temporal;
+	    boost::unordered_map<long unsigned int, real> total_mass;
```

Comment:
At line 45 of Particle.hpp, the particle property `mass` is declared as  `real`. Then, I think it is OK.